### DISCUSSION TITLE
佬们求助，我这创建机器人总是All connection attempts failed

### DISCUSSION THREAD

**Discussion by KennyYang0** [nonebot/discussions#432] [问答] [answered]
服务器已经重启过
可以与GitHub正常通信
没有做什么特殊的设置

**Comment by KennyYang0** [comment]
![屏幕截图 2024-01-26 170817](https://github.com/nonebot/discussions/assets/133932142/49e3d002-40a5-44d8-9e52-bd8190402bee)
yi'jing'yan'z已经严重到这种情况叻，创建第一步就失败

**Reply by yanyongyu** [reply, marked as answer]
你可以手动尝试一下请求 https://registry.nonebot.dev/adapters.json 看看能不能正常访问
